webapp/components/CommentList/CommentList.vue

```diff
@@ -36,8 +36,13 @@ export default {
     },
   },
   computed: {
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-08-07T08:33:43Z" title="Friday, August 7th 2020, 10:33:43 am +02:00">Aug 7, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -39,6 +39,14 @@ export default {
     postComments() {
       return (this.post && this.post.comments) || []
     },
```


This code is now superfluous, I think.